pandas/core/arrays/integer.py

```diff
@@ -573,6 +573,12 @@ def _reduce(self, name: str, skipna: bool = True, **kwargs):
 
         return result
 
+    def sum(self, skipna: bool = True, min_count: int = 0):
```

Comment:
probably should make the signature match PandasArray.sum etc

Reply:
Ok, and we still want to keep axis (even though it wouldn't be used here)?